mara_metabase/acl.py

```diff
@@ -38,6 +40,8 @@ def sync():
                     if email not in metabase_users:
                         print(client.post('/api/user', metabase_user))
                     else:
+                        if email in deactivated_metabase_users:
+                            print(client.put(f'/api/user/{metabase_users[email]}/reactivate', metabase_user))
```

Comment:
This needs the json?

Reply:
`deactivated_metabase_users` could also be a set, but I wanted to keep it consistent with the other dicts.

The next line needs to happen unconditionally, the group of a user might have changed upon re-invitation